deploy/helm/kubecf/defs.bzl

```diff
@@ -0,0 +1,8 @@
+def metadata_file_generator(name, file, operator_chart, visibility=None):
+  native.genrule(
+    name = name,
+    srcs = [],
+    outs = [file],
+    cmd = "echo 'operatorChartUrl: \"%s\"' > $@" % operator_chart,
```

Comment:
Prefer `"{}".format(...)` syntax in Python.

Reply:
Fixed, squashed and rebased :+1: